ecl/hql/hqlutil.cpp

```diff
@@ -5524,17 +5524,37 @@ static bool matchOption(unsigned & cur, unsigned max, const char * buffer, unsig
 IHqlExpression * extractCppBodyAttrs(unsigned lenBuffer, const char * buffer)
 {
     OwnedHqlExpr attrs;
+    bool isInComment = false;
     unsigned prev = '\n';
+
     for (unsigned i=0; i < lenBuffer; i++)
     {
         char next = buffer[i];
         switch (next)
         {
+        case '*':
+            if ('/' == prev)
```

Comment:
I think:
```
/* some comment */*outside comment
```
will cause isInComment to be spuriously marked true on the '*' proceeding 'outside'